pkg/hostagent/hostagent.go

```diff
@@ -466,6 +466,22 @@ func (a *HostAgent) startHostAgentRoutines(ctx context.Context) error {
 			return unmountMErr
 		})
 	}
+	if len(a.y.Volumes) > 0 {
+		a.onClose = append(a.onClose, func() error {
+			var unlockMErr error
+			for _, v := range a.y.Volumes {
+				vol, inspectErr := store.InspectVolume(v)
+				if inspectErr != nil {
+					unlockMErr = multierror.Append(unlockMErr, inspectErr)
+					continue
+				}
+				if unlockErr := vol.Unlock(); unlockErr != nil {
```

Comment:
For symmetry with the mounts it would be nice to provide logging for mounting/unmounting volumes too.

```console
$ l start default
[...]
INFO[0022] [hostagent] Mounting "/Users/jan" on "/Users/jan"
INFO[0022] [hostagent] Mounting "/tmp/lima" on "/tmp/lima"

$ l stop default
[...]
INFO[0000] [hostagent] Unmounting "/Users/jan"
INFO[0000] [hostagent] Unmounting "/tmp/lima"
```
There is no corresponding logging for volumes right now.

For mounts the logging happens in `m.Close()`. I'm not sure if the logging should happen in `vol.Unlock()` or explicitly here in the hostagent code. The rest of `pkg/store` is not emitting any logs because it is low-level code.